### DISCUSSION TITLE
Revoke Button confirmation pop-up

### DISCUSSION THREAD

**Discussion by CrystalDesignDR** [polymind-inc/acmebot#549] [Ideas]
Hi,
first off all, thanks for the awesome project.

While it is possible to restrict access for revoking certificates based on App Roles it still is kind of a danger zone.
Would it be possible / make sense to get a few more confirmations from a user when the Revoke button is clicked?

Example: 
Title: "Are you absolutely sure?"
Body description: "Revoking this certificate is an irreversible action. Please type _certificatename_ to confirm"
Input field: "" <- the name of the certificate must be entered to proceed
Delete Button

Example from GitHub (when you want to delete a repo):
![image](https://user-images.githubusercontent.com/48206301/209681624-6542d425-887a-482a-9c85-41ccb28443eb.png)

As a basic alternative a browser pop-up would be enough as well.

Related:
[Authentication - Restricting users](https://github.com/shibayan/keyvault-acmebot/discussions/441) and [App Role based authorization](https://github.com/shibayan/keyvault-acmebot/wiki/App-Role-based-authorization)

**Comment by shibayan** [comment]
Thanks for the feedback. I would like to add the implementation of a confirmation dialog. Issue created #555
